go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/trzsz/trzsz-ssh
 
-go 1.20
+go 1.21.4
```

Comment:
I use `go 1.20` for compatibility with Windows 7.

Reply:
> I use `go 1.20` for compatibility with Windows 7.
https://github.com/abakum/Win7
![2024-05-10_20-57-41](https://github.com/trzsz/trzsz-ssh/assets/18271863/b73af486-2a13-4a59-9196-3618187841a9)

Reply:
https://github.com/golang/go/issues/57003

Reply:
"Αμφισβητήστε τα πάντα" \8^)